spec.bs

```diff
@@ -347,6 +530,196 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
     1. Return the value of [=addModule success=].
   </div>
 
+  ## {{SharedStorageUrlWithMetadata}} and Reporting ## {#reporting}
+
+  A {{SharedStorageUrlWithMetadata}} {{/object}} is a [=dictionary=] containing a [=string=] representing a [=/URL=] and, optionally, a {{reportingMetadata}} {{/object}}.
```

Comment:
```suggestion
  {{SharedStorageUrlWithMetadata}} [=dictionaries=] contain a [=string=] representing a [=/URL=] and, optionally, a {{reportingMetadata}} {{/object}}.
```

SSUWM is not an object, it is _just_ a dictionary. There are a few more references below that reference SSUWM as an `{{/object}}`, can you get rid of those?